src/Sarif.Viewer.VisualStudio.UnitTests/Converters/CallTreeNodeToTextConverterTests.cs

```diff
@@ -148,7 +147,6 @@ public void CallTreeNodeToTextConverter_HandlesMessage()
                         },
                         PhysicalLocation = new PhysicalLocation
                         {
-                            Uri = new System.Uri(sourceFile),
```

Comment:
Why did this disappear, instead of being wrapped in a `FileLocation`?

Reply:
When you set the CallTreeNode.Location, there is a code path that ends up needing the IServiceProvider. I thought it best to just remove that property, since it's not involved in the test.

Reply:
Sure, if it's not needed for the test, this is fine.